ci-operator/step-registry/dpu-operator/e2e-tests-workflow/tests/dpu-operator-e2e-tests-workflow-tests-commands.sh

```diff
@@ -1,79 +1,96 @@
 #!/bin/bash
 
+echoerr() { echo "$@" 1>&2; }
+
+trigger_build() {
+	local pullnumber=$1
+	local token=$2
+	local endpoint=$3
+	local test_name=$4
+
+	local endpoint_resolve="${endpoint}:443:10.0.180.88"
+
+	curl -sik --resolve "$endpoint_resolve" "https://${endpoint}/job/${test_name}/buildWithParameters?token=${token}&pullnumber=${pullnumber}" | grep location: | tr -d '\r\n' | cut -d' ' -f2
+}
+
+
 wait_for_job_to_run() {
-	echo "Waiting for job to start..."
-	max_sleep_duration=432000  # Maximum sleep duration in seconds (5 days)
+	local endpoint=$2
+	echoerr "Waiting for job to start..."
+	# Maximum sleep duration in seconds (5 days)
+	max_sleep_duration=432000
 	sleep_counter=0
+
+	local endpoint_resolve="${endpoint}:443:10.0.180.88"
```

Comment:
``` local endpoint_resolve="${endpoint}:443:10.0.180.88"```
Is defined three times.

Reply:
agreed this variable doesn't change. It can probably be defined alongside the other static variables